react/features/notifications/components/NotificationsContainer.native.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import React from 'react';
+import { View } from 'react-native';
+import { connect } from 'react-redux';
+
+import AbstractNotificationsContainer, {
+    _abstractMapStateToProps,
+    type Props as AbstractProps
+} from './AbstractNotificationsContainer';
+import Notification from './Notification';
+import styles from './styles';
+
+type Props = AbstractProps & {
+
+    /**
+     * Any custom styling applied to the notifications container.
+     */
+    style: Object
+};
+
+/**
+ * Implements a React {@link Component} which displays notifications and handles
+ * automatic dismissmal after a notification is shown for a defined timeout
+ * period.
+ *
+ * @extends {Component}
+ */
+class NotificationsContainer
+    extends AbstractNotificationsContainer<Props> {
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
```

Comment:
Really a small thing, but if you use ```inheritdoc```, then it only makes sense to add a ```returns``` annotation if you want to overwrite the inherited value, which is not the case here, I guess.